.github/workflows/linux-cpu-x64-build.yml

```diff
@@ -68,7 +68,8 @@ jobs:
       - name: Install the python wheel and test dependencies
         run: |
           python3 -m pip install -r test/python/requirements.txt --user
-          python3 -m pip install -r test/python/requirements-cpu.txt --user
+          python3 -m pip install -r test/python/requirements-torch-cpu.txt --user
```

Comment:
Please put the requirements.txt files in different folders, and keep the filename part always being "requirements.txt", no extra words. So that [dependentbot ](https://github.com/dependabot) can recognize it.